components/library/clutter-gst/Makefile

```diff
@@ -15,11 +15,11 @@
 #
 
 BUILD_STYLE=configure
-BUILD_BITS=64
 include ../../../make-rules/shared-macros.mk
```

Comment:
BUILD_STYLE=configure is also superfluous because it's our default.

Reply:
done -  thanks